src/web/EmployerFavouritesWeb/Controllers/ApprenticeshipsParameterValidator.cs

```diff
@@ -0,0 +1,31 @@
+using System.Text.RegularExpressions;
```

Comment:
Should this be in a different location other than controllers folder

Reply:
👍 Moved to a validation folder.